test/e2e-tests.sh

```diff
@@ -49,24 +49,19 @@ function wait_for_leader_controller() {
   return 1
 }
 
-function enable_tag_header_based_routing() {
-  echo -n "Enabling Tag Header Based Routing"
-  kubectl patch cm config-network -n "${SYSTEM_NAMESPACE}" -p '{"data":{"tagHeaderBasedRouting":"Enabled"}}'
+function toggle_feature() {
+  local FEATURE="$1"
+  local STATE="$2"
+  local CONFIG="${3:-config-features}"
+  echo -n "Setting feature ${FEATURE} to ${STATE}"
+  kubectl patch cm "${CONFIG}" -n "${SYSTEM_NAMESPACE}" -p '{"data":{"'${FEATURE}'":"'${STATE}'"}}'
+  # We don't have a good mechanism for positive handoff so sleep :(
+  echo "Waiting 30s for change to get picked up."
+  sleep 30
```

Comment:
30 seems too much? Probably 5-10 range is enough?

Reply:
I dropped it to 10s.  I don't really like any answer here that isn't positive hand-off :(